src/modules/EpubView/EpubView.js

```diff
@@ -44,6 +44,9 @@ class EpubView extends Component {
   }
 
   componentWillUnmount() {
+    if (this.book) {
+      this.book.destroy()
+    }
     this.book = this.rendition = this.prevPage = this.nextPage = null
```

Comment:
Previously, `initBook` can not call `destroy()` because `this.book = null` on `componentWillUnmount`.

```js
  initBook() {
    const { url, tocChanged, epubInitOptions } = this.props
    if (this.book) {
      this.book.destroy()
    }
```